src/registrar/templates/includes/finish_profile_form.html

```diff
@@ -0,0 +1,89 @@
+{% extends 'includes/profile_form.html' %}
+
+{% load static url_helpers %}
+{% load field_helpers %}
+
+{% block profile_header %}
+<h1>Finish setting up your profile</h1>
+{% endblock profile_header %}
+
+{% block profile_blurb %}
+<p>
+    We <a class="usa-link" href="{% public_site_url 'domains/requirements/#what-.gov-domain-registrants-must-do' %}"  target="_blank">require</a> 
```

Comment:
Now that we've linkified all headers. Can we change this link to https://get.gov/domains/requirements/#keep-your-contact-information-updated so it specifically points to that section about contact info?